Packages/Signal.lua

```diff
@@ -0,0 +1 @@
+return require(script.Parent._Index["sleitnick_signal@1.0.1"]["signal"])
```

Comment:
The Packages folder should be added to the .gitignore, dependencies should generally not be in the repository.

Reply:
This makes sense, though we will need to have the CI workflow run ``wally install``.

Reply:
We will also need to change the testing scripts/formatting to account for the addition of the Packages folder.

Reply:
Edit: Packages has been completely removed. I am leaving the comment unresolved so you can verify this fix properly address your comment